services/cloudrun-service/pkg/cloudrun/cloudrun.go

```diff
@@ -0,0 +1,177 @@
+/*This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the Expat(MIT) License as published by
+the Free Software Foundation.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+MIT License for more details.
+
+You should have received a copy of the MIT License
+along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
+
+Copyright 2023 freiheit.com*/
+
+package cloudrun
+
+import (
+	"context"
+	"fmt"
+
+	"google.golang.org/api/run/v1"
+)
+
+const (
+	serviceLocationLabel       = "cloud.googleapis.com/location"
+	operationIdAnnotation      = "run.googleapis.com/operation-id"
+	serviceReady               = "Ready"
+	serviceConfigurationsReady = "ConfigurationsReady"
+	serviceRoutesReady         = "RoutesReady"
+)
+
+var (
+	runService *run.APIService
+)
+
+// Possible values are True, False or Unknown
+type ServiceConditions struct {
+	Ready              string
```

Comment:
if they can't be `bool`, they should use a "enum":
https://builtin.com/software-engineering-perspectives/golang-enum
something like
```
type ReadyState int

const (// Declare related constants for each weekday starting with index 1
	Ready    ReadyState = iota + 1
	NotReady
        Unknown
// ...
```